Assets/MRTK/SDK/Features/Input/Handlers/Constraints/TransformConstraint.cs

```diff
@@ -13,19 +13,6 @@ public abstract class TransformConstraint : MonoBehaviour
     {
         #region Properties
 
-        [SerializeField]
-        [Tooltip("Transform being constrained. Defaults to the object of the component.")]
-        private Transform targetTransform = null;
-
-        /// <summary>
-        /// Transform that we intend to apply constraints to
-        /// </summary>
-        public Transform TargetTransform
```

Comment:
Can this be left around with an obsolete tag stating that it wasn't used and will be removed in a future release? That way we don't break any existing code that references this public property. Can just be an auto-property so we don't need an explicit backing field.